change/@uifabric-experiments-2020-05-27-17-52-49-u-nebhatna-accessibilityIssues.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "minor accessibility changes",
```

This should include the component name since it goes in the package-level release notes.
```suggestion
  "comment": "SelectedItemsList: minor accessibility changes",
```

---

thanks, did not know. will add this